src/test.js

```diff
@@ -378,7 +378,7 @@ Test.prototype = {
 					function( error ) {
 						message = "Promise rejected " +
 							( !phase ? "during" : phase.replace( /Each$/, "" ) ) +
-							" " + test.testName + ": " + ( error.message || error );
+							" \"" + test.testName + "\": " + ( error && error.message );
```

Comment:
Yeah, it should. Late night coding mistake when thinking about how short-circuit evaluation works.